Makefile

```diff
@@ -71,7 +71,7 @@ stop-localnet-silent:
 	@kurtosis enclave stop lambdanet >/dev/null 2>&1 || true
 	@kurtosis enclave rm lambdanet --force >/dev/null 2>&1 || true
 
-HIVE_REVISION := 3be4465a45c421651d765f4a28702962567b40e6
```

Comment:
remove this change plz, since we now have https://github.com/lambdaclass/lambda_ethereum_rust/pull/1007